plugins/woocommerce/tests/legacy/unit-tests/core/template-cache.php

```diff
@@ -43,7 +43,9 @@ public function test_wc_get_template_part() {
 
 		// Check individual templates.
 		foreach ( $templates as $cache_key => $template ) {
-			$this->assertEquals( $template, wp_cache_get( $cache_key, 'woocommerce' ) );
+			// The cache contents may be tokenized and we need to untokenize it for the assertion.
+			$from_cache = wc_untokenize_path( wp_cache_get( $cache_key, 'woocommerce' ), wc_get_path_define_tokens() );
```

Comment:
This change comes because of the difference in execution context. Running from `/Users/obliviousharmony/Code/woocommerce/woocommerce/plugins/woocommerce` will not tokenize anything because there are no token paths present. Running from `/var/www/html/wp-content/plugins/woocommerce` inside of `wp-env`, however, will be tokenized because it's in a real WordPress installation.